testsuite/features/support/env.rb

```diff
@@ -475,6 +479,13 @@ def process_code_coverage
   skip_this_scenario if scenario.location.file.include? 'slemicro'
 end
 
+Before('@skip_for_sle_micro_ssh_minion') do |scenario|
+  micros = %w[slemicro51_ssh_minion slemicro52_ssh_minion slemicro53_ssh_minion slemicro54_ssh_minion]
+  micros.each do |ssh_minion|
+    skip_this_scenario if scenario.location.file.include? ssh_minion
+  end
+end
+
```

Comment:
```suggestion Before('@skip_for_sle_micro_ssh_minion') do |scenario|
   sle_micro_ssh_nodes = %w[slemicro51_ssh_minion slemicro52_ssh_minion slemicro53_ssh_minion slemicro54_ssh_minion]
   current_feature_node = scenario.location.file.split(%r{(\.feature|\/)})[-2] # We can extract this in a method, i.e. get_feature_name(scenario)
   skip_this_scenario if sle_micro_ssh_nodes.include? current_feature_node
 end
```

Reply:
This does not work:

```ruby
Before('@skip_for_sle_micro_ssh_minion') do |scenario|
  sle_micro_ssh_nodes = %w[slemicro51_ssh_minion slemicro52_ssh_minion slemicro53_ssh_minion slemicro54_ssh_minion]
  current_feature_node = scenario.location.file.split(%r{(\.feature|\/)})[-2] # We can extract this in a method, i.e. get_feature_name(scenario)
  puts sle_micro_ssh_nodes.include? current_feature_node
  puts current_feature_node
  skip_this_scenario if sle_micro_ssh_nodes.include? current_feature_node
end
```

 ```cucumber
Feature: doms tests

@skip_for_sle_micro_ssh_minion
  Scenario: doms tests
    When I run doms tests
```

```bash
uyuni-bv-master-ctl:~/spacewalk/testsuite # cucumber features/slemicro51_ssh_minion_smoke_tests.feature 
(...)
Initializing a twopence node for 'server'.
Host 'server' is alive with determined hostname uyuni-bv-master-srv and FQDN uyuni-bv-master-srv.mgr.suse.de
Node: uyuni-bv-master-srv, OS Version: 15.4, Family: opensuse-leap
Activating HTTP API
Using the default profile...
Feature: doms tests

  @skip_for_sle_micro_ssh_minion
  Scenario: doms tests    # features/slemicro51_ssh_minion_smoke_tests.feature:4
      This scenario ran at: 2023-10-02 10:59:12 +0200
false
slemicro51_ssh_minion_smoke_tests
(...)
    When I run doms tests # features/step_definitions/common_steps.rb:536
      This scenario took: 1 seconds

1 scenario (1 passed)
1 step (1 passed)
0m0.589s
```
It should at least return true.

Reply:
Sure thing, we will need to remove "_smoke_tests" too :) I fixed my suggestion above.

Reply:
I can confirm that this does work now:

```bash
uyuni-bv-master-ctl:~/spacewalk/testsuite # cucumber features/slemicro51_ssh_minion_smoke_tests.feature
(...)
Capybara APP Host: https://uyuni-bv-master-srv.mgr.suse.de:8888
Initializing a twopence node for 'server'.
Host 'server' is alive with determined hostname uyuni-bv-master-srv and FQDN uyuni-bv-master-srv.mgr.suse.de
Node: uyuni-bv-master-srv, OS Version: 15.4, Family: opensuse-leap
Activating HTTP API
Using the default profile...
Feature: doms tests

  @skip_for_sle_micro_ssh_minion
  Scenario: doms tests    # features/slemicro51_ssh_minion_smoke_tests.feature:4
      This scenario ran at: 2023-10-05 16:49:13 +0200
true
slemicro51_ssh_minion
    When I run doms tests # features/step_definitions/common_steps.rb:536
      This scenario took: 0 seconds

1 scenario (1 skipped)
1 step (1 skipped)
0m0.004s
```